src/System.Private.CoreLib/src/System/Reflection/RuntimePropertyInfo.cs

```diff
@@ -224,6 +224,7 @@ private RuntimeType ReflectedTypeInternal
 
         public override Module Module { get { return GetRuntimeModule(); } }
         internal RuntimeModule GetRuntimeModule() { return m_declaringType.GetRuntimeModule(); }
+        public override bool IsCollectible => m_getterMethod.IsCollectible || m_setterMethod.IsCollectible;
```

Comment:
For completeness, it may be nice to for-loop over `m_otherMethod` array like what's done in other places in this file.